user-manual/import-export/csv-export.rst

```diff
@@ -344,6 +344,27 @@ will allow you to "unzip" a ZIP file.
 
    * https://wiki.accesstomemory.org/Resources/CSV_templates
 
+**To include digital objects with your CSV export:**
+
+Some AtoM sites are configured to enable users to include attached digital
```

Comment:
It would probably be good to add a link in here somewhere to the settings page, so administrators just learning about this functionality know where to go to configure it.

Reply:
In line 349, make "configured" a link to https://www.accesstomemory.org/en/docs/2.7/user-manual/administer/settings/#clipboard-settings.